lib/coinbase/staking_reward.rb

```diff
@@ -82,5 +62,18 @@ def inspect
     def self.stake_api
       Coinbase::Client::StakeApi.new(Coinbase.configuration.api_client)
     end
+
+    def self.list_page(network_id, asset_id, address_ids, start_time, end_time, page, format)
```

Comment:
Would be preferable to make these private class methods via `private_class_method` or the `class << self` syntax.

I'm probably going to do a pass on all classes for this though!